main/2020/CG-04-14.md

```diff
@@ -39,4 +39,4 @@ Installation is required, see the calendar invite.
 
 ## Meeting Notes
 
-To be added after the meeting.
+Ross Tate presenting slides for Stack and Control Primitives [(pdf)](presentations/2020-04-14-tate-stack-primitives.pdf) [(pptx)](presentations/2020-04-14-tate-stack-primitives.pptx)
```

Comment:
Interesting, I wonder what happened to the notes for this meeting...? I'll check.